save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkContestProjectController.kt

```diff
@@ -83,6 +82,49 @@ class LnkContestProjectController(
             lnkContestProject.toContestResult(score)
         }
 
+    /**
+     * @param contestName
+     * @param authentication
+     * @return list of user's [Project]s that can participate in contest with name [contestName]
+     */
+    @GetMapping("/{contestName}/avaliable")
```

Comment:
```suggestion
    @GetMapping("/{contestName}/eligible-projects")
```
Maybe something like this? At first I thought that this endpoint would return Boolean whether the caller can participate in the contest

Reply:
oh ok let it be so

Reply:
Done